### ISSUE TITLE
NameError: uninitialized constant ActiveRecord::ConnectionAdapters::OracleEnhancedColumn::FALSE_VALUES

### ISSUE THREAD

**Issue by workmonitored** [rsim/oracle-enhanced#893]
I am running on windows ruby 2.3.0. I have installed the active record and adapter. When I try to connect I got weird behaviour such as rspec terminating with success without any messages so I tried the debugging recommended on the first page. The ruby_oci8 works without problems for me. However when I try the command for establishing a connection for the active record through irb. I am getting this error. 

>> require 'rubygems'
=> false
>> gem 'activerecord'
=> false
>> gem 'activerecord-oracle_enhanced-adapter'
=> false
>> require 'active_record'
=> false
< => "oracle_enhanced", :database => "something",:username => "somebody",:password => "apassword")
NameError: uninitialized constant ActiveRecord::ConnectionAdapters::OracleEnhancedColumn::FALSE_VALUES
        from C:/web/Ruby23/lib/ruby/gems/2.3.0/gems/activerecord-oracle_enhanced-adapter-1.6.7/lib/active_record/connection_adapters/oracle_enhan..olumn.rb:7:in `<class:OracleEnhancedColumn>'
        from C:/web/Ruby23/lib/ruby/gems/2.3.0/gems/activerecord-oracle_enhanced-adapter-1.6.7/lib/active_record/connection_adapters/oracle_enhan..olumn.rb:3:in `<module:ConnectionAdapters>'
        from C:/web/Ruby23/lib/ruby/gems/2.3.0/gems/activerecord-oracle_enhanced-adapter-1.6.7/lib/active_record/connection_adapters/oracle_enhan..olumn.rb:2:in `<module:ActiveRecord>'
        from C:/web/Ruby23/lib/ruby/gems/2.3.0/gems/activerecord-oracle_enhanced-adapter-1.6.7/lib/active_record/connection_adapters/oracle_enhan..olumn.rb:1:in `<top (required)>'
        from C:/web/Ruby23/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
        from C:/web/Ruby23/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
        from C:/web/Ruby23/lib/ruby/gems/2.3.0/gems/activerecord-oracle_enhanced-adapter-1.6.7/lib/active_record/connection_adapters/oracle_enhan..dapter.rb:39:in `<top (required)>'
        from C:/web/Ruby23/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
        from C:/web/Ruby23/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
        from C:/web/Ruby23/lib/ruby/gems/2.3.0/gems/activerecord-5.0.0/lib/active_record/connection_adapters/connection_specification.rb:174:in `

        from C:/web/Ruby23/lib/ruby/gems/2.3.0/gems/activerecord-5.0.0/lib/active_record/connection_handling.rb:53:in `establish_connection'
        from (irb):5
        from C:/web/Ruby23/bin/irb.cmd:19:in `<main>'

My installed bundles are

Gems included by the bundle:
  * activemodel (5.0.0)
  * activerecord (5.0.0)
  * activerecord-oracle_enhanced-adapter (1.6.7)
  * activesupport (5.0.0)
  * addressable (2.4.0)
  * arel (7.0.0)
  * bundler (1.12.5)
  * capybara (2.7.1)
  * concurrent-ruby (1.0.2)
  * diff-lcs (1.2.5)
  * htmlentities (4.3.4)
  * httparty (0.13.7)
  * i18n (0.7.0)
  * json (1.8.3)
  * mime-types (3.1)
  * mime-types-data (3.2016.0521)
  * mini_portile2 (2.1.0)
  * minitest (5.9.0)
  * multi_xml (0.5.5)
  * nokogiri (1.6.8)
  * os (0.9.6)
  * pkg-config (1.1.7)
  * rack (1.6.4)
  * rack-protection (1.5.3)
  * rack-test (0.6.3)
  * rspec (3.5.0)
  * rspec-core (3.5.1)
  * rspec-expectations (3.5.0)
  * rspec-mocks (3.5.0)
  * rspec-retry (0.4.5)
  * rspec-support (3.5.0)
  * ruby-oci8 (2.2.2)
  * sinatra (1.4.7)
  * thread_safe (0.3.5)
  * tilt (2.0.5)
  * tzinfo (1.2.2)
  * xpath (2.0.0)

**Comment by thiaguerd** [comment]
try put this on your Gemfile
````ruby
gem 'activerecord-oracle_enhanced-adapter', git: 'git@github.com:rsim/oracle-enhanced.git', branch: 'rails5'
````

**Comment by workmonitored** [comment]
Done but I am getting an error, it can't find the gem

```
>> gem 'activerecord-oracle_enhanced-adapter'
Gem::LoadError: Could not find 'activerecord-oracle_enhanced-adapter' (>= 0) among 47 total gem(s)
Checked in 'GEM_PATH=C:/Users/first.last/.gem/ruby/2.3.0;`C:/web/Ruby23/lib/ruby/gems/2.3.0', execute `gem env` for more information
        from C:/web/Ruby23/lib/ruby/2.3.0/rubygems/dependency.rb:319:in `to_specs'
        from C:/web/Ruby23/lib/ruby/2.3.0/rubygems/dependency.rb:328:in `to_spec'
        from C:/web/Ruby23/lib/ruby/2.3.0/rubygems/core_ext/kernel_gem.rb:65:in `gem'
        from (irb):3
```

It looks like it is installed here

 Directory of C:\web\Ruby23\lib\ruby\gems\2.3.0\bundler\gems\oracle-enhanced-ef4f68fdf96b

**Comment by thiaguerd** [comment]
you run 
````
bundler install
````
?

**Comment by workmonitored** [comment]
I installed, using
```

C:\web\domo\domo_eq\integration_tests>bundle install
Fetching git://github.com/rsim/oracle-enhanced.git
Fetching gem metadata from https://rubygems.org/
Fetching version metadata from https://rubygems.org/
Resolving dependencies....
Using concurrent-ruby 1.0.2
Using i18n 0.7.0
Using minitest 5.9.0
Using thread_safe 0.3.5
Using arel 7.0.0
Using rspec-support 3.5.0
Using diff-lcs 1.2.5
Installing ruby-oci8 2.2.2
Using ruby-plsql 0.6.0
Using addressable 2.4.0
Using bundler 1.12.5
Using mime-types-data 3.2016.0521
Using mini_portile2 2.1.0
Using pkg-config 1.1.7
Using rack 1.6.4
Using htmlentities 4.3.4
Using json 1.8.3
Using multi_xml 0.5.5
Using os 0.9.6
Using tilt 2.0.5
Using tzinfo 1.2.2
Using rspec-core 3.5.1
Using rspec-expectations 3.5.0
Using rspec-mocks 3.5.0
Using mime-types 3.1
Using nokogiri 1.6.8
Using rack-test 0.6.3
Using rack-protection 1.5.3
Using httparty 0.13.7
Using activesupport 5.0.0
Using rspec-retry 0.4.5
Using rspec 3.5.0
Using xpath 2.0.0
Using sinatra 1.4.7
Using activemodel 5.0.0
Using capybara 2.7.1
Using activerecord 5.0.0
Using activerecord-oracle_enhanced-adapter 1.6.7 from git://github.com/rsim/oracle-enhanced.git (at rails5@ef4f68f)
Bundle complete! 12 Gemfile dependencies, 38 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.

C:\web\domo\domo_eq\integration_tests>bundle show activerecord-oracle_enhanced-adapter
C:/web/Ruby23/lib/ruby/gems/2.3.0/bundler/gems/oracle-enhanced-ef4f68fdf96b
```

**Comment by thiaguerd** [comment]
so all is ok
what operation with database is not working?

**Comment by workmonitored** [comment]
connection hangs up and rspec terminates. I started troubleshooting and figured out ruby terminates when I run the establish connection.  I run the irb command for ruby_oci8 (works), I get this uninitialized constant error when I run establish connection using oracle enhanced adapter. I am guessing this is the cause. Question for windows, do I need ruby developer extension to install or run this ?

**Comment by thiaguerd** [comment]
don't execute the comand
````bash
irb
````
execute
````bash
rails console
`````
on location of your rails project

**Comment by workmonitored** [comment]
I don't have a rails project. It is a project with rspec tests. One of the gemfile dependencies is the adapter. When I run the rspec, it fails on the connection to the db but I cannot tell what's failing because it just says rspec terminated with success and ruby terminated with success. I then narrowed it to the line where connection is made to the database. However with no diagnostic output, it is hard to figure out why failing so, I started running the irb tests found on the project site to see if I can see the error.
